src/Apps/ArtQuiz/Hooks/DislikeArtworkMutation.ts

```diff
@@ -0,0 +1,35 @@
+import { Environment, commitMutation, graphql } from "react-relay"
+import {
+  DislikeArtworkMutation,
+  DislikeArtworkInput,
+  DislikeArtworkMutation$data,
+} from "__generated__/DislikeArtworkMutation.graphql"
+
+export const DislikeArtwork = (
```

Comment:
Can use our `useMutation` hook ([here's an example](https://github.com/artsy/force/blob/main/src/Apps/Settings/Routes/DeleteAccount/useDeleteAccount.ts) of how to use)

Reply:
Oh nice. Will take a look.